mriqc/config.py

```diff
@@ -435,10 +435,6 @@ def init(cls):
             ignore_paths = [
                 # Ignore folders at the top if they don't start with /sub-<label>/
                 re.compile(r"^(?!/sub-[a-zA-Z0-9]+)"),
-                # Ignore all modality subfolders, except for func/ or anat/
-                re.compile(
-                    r"^/sub-[a-zA-Z0-9]+(/ses-[a-zA-Z0-9]+)?/(?!func|anat)"
-                ),
                 # Ignore all files, except for the supported modalities
```

Comment:
Now that you've got a test, you could try: `r"^/sub-[a-zA-Z0-9]+(/ses-[a-zA-Z0-9]+)?/(beh|dwi|fmap|pet|perf|meg|eeg|ieeg|micr|nirs)/"`
                ),

Reply:
Added. I dropped the trailing slash because it makes indexing more efficient after my PR.